packages/util/type-utils.test-d.ts

```diff
@@ -10,75 +10,79 @@ import type {
 	SanitizeKeyPrefixLeadingNumber,
 	SanitizeKeyRemoveDisallowedChar,
 	SanitizeKeys,
-} from "./type-utils.d.ts";
+} from "./type-utils.js";
 
 // SanitizeKeyPrefixLeadingNumber
 // adds a leading _ if it starts with a number
-expectType<SanitizeKeyPrefixLeadingNumber<"1234">>("_1234");
-expectType<SanitizeKeyPrefixLeadingNumber<"01234">>("_01234");
+expect<SanitizeKeyPrefixLeadingNumber<"1234">>().type.toBe<"_1234">();
+expect<SanitizeKeyPrefixLeadingNumber<"01234">>().type.toBe<"_01234">();
```

Comment:
`expect`() can compare two types. The utilities tested in this file are working with types, not values. So I refactored the tests to compare two types instead a type and inferred type of an expression.